entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/vocabulary/WebEntityFields.java

```diff
@@ -45,7 +46,7 @@
 	public static final String BROAD_MATCH = "broadMatch";
 	public static final String NARROW_MATCH = "narrowMatch";
 	public static final String RELATED_MATCH = "relatedMatch";
-	public static final String IN_SCHEME = "inScheme";
+	public static final String IN_SCHEMA = "inSchema";
```

Comment:
please reverts the correct name of the field is inScheme

Reply:
Done.